src/Draco.Compiler/Api/Syntax/ElseClauseSyntax.cs

```diff
@@ -0,0 +1,11 @@
+namespace Draco.Compiler.Api.Syntax;
+
+public partial class ElseClauseSyntax
+{
+    /// <summary>
+    /// Returns <see langword="true"/> when the else clause is followed by an if expression.
+    /// </summary>
+    public bool IsElseIf => this.Expression is StatementExpressionSyntax statementExpression
```

Comment:
Please check if this works in expression context too

Reply:
Added tests + fixed this.